app/src/main/java/movierating/services/ReviewServiceImpl1.java

```diff
@@ -35,7 +35,7 @@ public void addReview(String userName, String movieName, int rating) throws Movi
         if (movieRepository.getMovies() != null && movieRepository.getMovies().containsKey(movieName))
             movie = this.movieRepository.getMovies().get(movieName);
         else {
-            throw new MovieNotReleasedException("Movie is not yet Released!!");
+            throw new MovieNotReleasedException();
```

Comment:
I think movie name should be passed as an argument to  `MovieNotReleasedException`. What do you think?

Reply:
I thought about it. Didn't seem necessary to me. But added anyway.